backend/b2/upload.go

```diff
@@ -124,6 +123,14 @@ func (f *Fs) newLargeUpload(ctx context.Context, o *Object, in io.Reader, src fs
 		request.Info = map[string]string{
 			timeKey: timeString(modTime),
 		}
+		// Custom upload headers - remove header prefix since they are sent in the body
```

Comment:
Can't put a comment in the right place, but this above

```go
opts: rest.Opts{
}
```

Should have

```
Options:    options,
```

This is how the non metadata options get set in the request.

Reply:
Agreed, although I think that the metadata options, that we now set in the body, should be removed from the list, otherwise they are [added to the HTTP headers](https://github.com/rclone/rclone/blob/master/lib/rest/rest.go#L289). Does that sound right to you, @ncw?

Reply:
It sounds right to me, so I went ahead with it.

Reply:
Looks good :-)